tests/test_tui_package_contract.py

```
@@ -0,0 +1,247 @@
+        return (
+            "cmux-tui-linux-arm64"
+            if machine in {"aarch64", "arm64"}
+            else "cmux-tui-linux-x64"
+        )
+    if system == "darwin":
+        return (
+            "cmux-tui-darwin-x64"
+            if machine in {"x86_64", "amd64"}
+            else "cmux-tui-darwin-arm64"
+        )
+    raise RuntimeError(f"unsupported test host: {system}-{machine}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)